app/models/reports/facility_progress_dimension.rb

```diff
@@ -0,0 +1,33 @@
+module Reports
+  class FacilityProgressDimension
```

Comment:
I was struggling to get a handle on all the different facets of how we display the data, and breaking it out to a tiny value object helped to make things more clear.